api/src/main/java/com/bjs/bjsapi/config/ClassificationInformationConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.bjs.bjsapi.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.IOException;
+import java.net.URL;
+
+@Configuration
+@ConfigurationProperties(prefix = "api.calculation")
+public class ClassificationInformationConfig {
```

Comment:
Maybe this class could be combined with the `CalculationInformationConifg` class.